package.json

```diff
@@ -10,25 +10,49 @@
     "fullstack"
   ],
   "dependencies": {
-    "@reshuffle/code-transform": "0.1.0",
+    "@apollo/react-hooks": "^3.1.3",
+    "@apollo/react-testing": "^3.1.3",
+    "@fullcalendar/core": "^4.3.1",
+    "@fullcalendar/daygrid": "^4.3.0",
+    "@fullcalendar/interaction": "^4.3.0",
+    "@fullcalendar/react": "^4.3.0",
+    "@material-ui/core": "^4.7.2",
+    "@reshuffle/code-transform": "^0.1.0",
     "@reshuffle/db": "^0.5.5",
-    "@reshuffle/fetch-runtime": "0.1.0",
-    "@reshuffle/local-proxy": "^0.3.1",
+    "@reshuffle/fetch-runtime": "^0.1.0",
+    "@reshuffle/local-proxy": "^0.3.4",
     "@reshuffle/passport": "^0.2.0",
     "@reshuffle/react-auth": "^0.1.7",
-    "@reshuffle/server-function": "0.2.2",
+    "@reshuffle/server-function": "^0.2.2",
+    "apollo": "^2.21.2",
+    "apollo-boost": "^0.4.7",
+    "apollo-cache-inmemory": "^1.6.3",
+    "apollo-client": "^2.6.4",
+    "apollo-link-http": "^1.5.16",
+    "apollo-server": "^2.9.13",
+    "apollo-server-express": "^2.9.13",
     "bootstrap": "^4.4.0",
+    "graphql": "^14.5.8",
+    "graphql-tag": "^2.10.1",
+    "luxon": "^1.21.3",
+    "nanoid": "^2.1.8",
+    "node-sass": "^4.13.0",
     "react": "^16.12.0",
     "react-bootstrap": "^1.0.0-beta.12",
     "react-dom": "^16.12.0",
-    "react-scripts": "^3.2.0"
+    "react-hook-form": "^3.29.2",
+    "react-pose": "^4.0.10",
+    "react-scripts": "^3.2.0",
+    "react-test-renderer": "^16.12.0",
+    "theme": "^0.1.0"
   },
   "scripts": {
     "start": "cross-env EXTEND_ESLINT=true react-scripts start",
     "build": "cross-env EXTEND_ESLINT=true react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint backend --fix && eslint src --fix"
+    "lint": "eslint backend --fix && eslint src --fix && stylelint 'src/**/*.{css,scss}'",
+    "check-prettier": "prettier --ignore-path .gitignore -c '**/*.{js,jsx}'"
```

Comment:
Please also include the `check-prettier` stage in `.circleci/config.yml` similar to https://github.com/reshufflehq/twitter-sentiment-example/commit/f789f2bae5a356fe95b192dd06a1ab9e8fd21532.

Reply:
Done

Reply:
In addition to defining the job it needs to be added to the `commit` workflow.

Reply:
Thank you for the link to document!
It's clear now.
I added `check-perttier` to the workflow